uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/document-list.js

```diff
@@ -113,18 +113,15 @@ define([
             };
 
             $scope.filterParams = {
-                contactId: null,
-                documentStatus: [],
-                userRole: $state.params.userRole || {
-                    field: null,
-                    isEqual: null
-                },
-                dateRange: {
-                    from: null,
-                    until: null
-                },
-                due: null,
-                assignmentType: []
+              contactId: null,
```

Comment:
Since document-list.js and task-list.js using same $scope.filterParams. We should fetch it from a function to reduce redundancy. It can be from a service/controller function/constant.